CS157A-Team18/CS157A-Team18
# [D-44] Migrate from Config File to .env File

Changes:
- Migrate to using .env file instead of config files

To Test:
1. Pull this branch
2. On the Google Drive root directory, there is an `.env` file. Make sure this is updated with the `.env` file in your local **root directory**. 
3. There will also be a folder called `React .env` on the Google Drive. Make sure this is placed under `/client`.
4. In the root directory, do `npm start`.
5. Open a new terminal and do `npm start` in the `/client` directory.
6. When the webpage opens up in the browser, type in a wrong username and password. You should get the following result in the console:
![image](https://user-images.githubusercontent.com/36905285/69506585-225bdf00-0ee4-11ea-8bf6-cfb8739890fe.png)
